### PR TITLE
Use FUNCTION_NAME as service id on GCF

### PR DESCRIPTION
**Pull request by matthewloring**

### CODE REVIEW

**Comment by matthewloring** [comment]
Before: 
screen shot 2017-06-06 at 5 56 55 pm

--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
--------------------------------------------------------------------------------
After:
screen shot 2017-06-06 at 6 59 14 pm

**Comment by ofrobots** [comment]
@swalkowski: Is there a concept of 'versions' for functions?

File: src/agent/debuglet.js [hunk 1]
```diff
@@ -146,8 +146,10 @@ Debuglet.normalizeConfig_ = function(config) {
   var envConfig = {
     logLevel: process.env.GCLOUD_DEBUG_LOGLEVEL,
     serviceContext: {
-      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME,
-      version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION,
+      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME ||
+        process.env.FUNCTION_NAME,
+      version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION ||
+        'unversioned',
```
Comment (ofrobots) [inline src/agent/debuglet.js]: When the app is running locally, we will be providing a serviceContext with `service` set to undefined and `version` set to `unversioned`. How would this display in the UI?

Comment (matthewloring) [inline src/agent/debuglet.js]: It should display the command used to start the process as is done today.

**Comment by swalkowski** [comment]
@ofrobots: No, functions are unversioned at the moment.

File: src/agent/debuglet.js [hunk 2]
```diff
@@ -146,8 +146,10 @@ Debuglet.normalizeConfig_ = function(config) {
   var envConfig = {
     logLevel: process.env.GCLOUD_DEBUG_LOGLEVEL,
     serviceContext: {
-      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME,
-      version: process.env.GAE_VERSION || process.env.GAE_MODULE_VERSION,
+      service: process.env.GAE_SERVICE || process.env.GAE_MODULE_NAME ||
```
Comment (DominicKramer) [inline src/agent/debuglet.js]: Is there a known situation where `GAE_SERVICE` or `GAE_MODULE_NAME` are intentionally set to the empty string.  I don't think this is the case, but I just want to make sure.

File: src/agent/debuglet.js (hunk 1 above)
Comment (matthewloring) [inline src/agent/debuglet.js]: Whoops, looks like that was not the case. Fixed.

File: src/agent/debuglet.js (hunk 2 above)
Comment (matthewloring) [inline src/agent/debuglet.js]: Not to my knowledge.

**Pull request merged**